test/e2e/usersignup_test.go

```diff
@@ -109,6 +109,42 @@ func (s *userSignupIntegrationTest) TestAutomaticApproval() {
 	})
 }
 
+func (s *userSignupIntegrationTest) TestProvisionToOtherClusterWhenOneIsFull() {
+	s.T().Run("set per member clusters max number of users for both members and expect that users will be provisioned to the other member when one is full", func(t *testing.T) {
+		// given
+		var memberLimits []test.PerMemberClusterOption
+		toolchainStatus, err := s.hostAwait.WaitForToolchainStatus(wait.UntilToolchainStatusHasConditions(ToolchainStatusReadyAndUnreadyNotificationNotCreated()...))
+		require.NoError(t, err)
+		for _, m := range toolchainStatus.Status.Members {
+			if s.memberAwait.ClusterName == m.ClusterName {
+				memberLimits = append(memberLimits, test.PerMemberCluster(s.memberAwait.ClusterName, m.UserAccountCount+1))
+			} else if s.member2Await.ClusterName == m.ClusterName {
+				memberLimits = append(memberLimits, test.PerMemberCluster(s.member2Await.ClusterName, m.UserAccountCount+1))
+			}
+		}
+		require.Len(s.T(), memberLimits, 2)
+
+		s.hostAwait.UpdateHostOperatorConfig(test.AutomaticApproval().Enabled().MaxUsersNumber(0, memberLimits...))
+
+		// when
+		_, mur1 := s.createAndCheckUserSignup(false, "multimember-1", "multi1@redhat.com", nil, ApprovedAutomatically()...)
+		// we need to sleep one second to create UserSignup with different creation time
+		time.Sleep(time.Second)
```

Comment:
This is actually not needed in this use-case - I used this hack to know which UserSignup won't be approved/provisioned and will end-up in the waiting list. But since we do expect that both of them should be provisioned, we don't need to put this time gap between them:
```suggestion
```

Reply:
Ah, didn't know that. Thanks for clarifying